typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -256,14 +267,22 @@ export class ContractVerifier {
     chain: ChainName,
     input: ContractVerificationInput,
     verificationLogger: Logger,
-  ): Promise<{ proxyGuid: any }> {
+  ): Promise<string> {
     try {
-      return this.submitForm(
+      const proxyGuid = await this.submitForm(
```

Comment:
nit: i find it confusing that two different sounding functions (this.markProxy and this.submitForm) return a variable with the same name (proxyGuid)

Reply:
`submitForm()` only returns the response for whatever the request is (in this case, `ExplorerApiActions.VERIFY_PROXY`).

In the context of `markProxy()`, this is always `proxyGuid` if the response returns a 200